test/ParallelIOTest.cpp

```diff
@@ -382,4 +382,93 @@ TEST_CASE( "hzdr_adios_sample_content_test", "[parallel][adios1]" )
         return;
     }
 }
+
+void
+close_iteration_test( std::string file_ending )
+{
+    int __mpi_rank{ -1 }, __mpi_size{ -1 };
+    MPI_Comm_rank( MPI_COMM_WORLD, &__mpi_rank );
+    MPI_Comm_size( MPI_COMM_WORLD, &__mpi_size );
+    unsigned mpi_rank{ static_cast< unsigned >( __mpi_rank ) },
+        mpi_size{ static_cast< unsigned >( __mpi_size ) };
```

Comment:
Double `__` prefixes are reserved for the compiler & stdlib:
```suggestion
    int i_mpi_rank{ -1 }, i_mpi_size{ -1 };
    MPI_Comm_rank( MPI_COMM_WORLD, &i_mpi_rank );
    MPI_Comm_size( MPI_COMM_WORLD, &i_mpi_size );
    unsigned mpi_rank{ static_cast< unsigned >( i_mpi_rank ) },
        mpi_size{ static_cast< unsigned >( i_mpi_size ) };
```